tools/setupResourceScripts/setup_tdr_resources.py

```diff
@@ -289,16 +288,23 @@ def find_dataset(dataset):
     return find_dataset
 
 
+def delete_dataset_snapshots(clients, dataset_id):
```

Comment:
Just a sanity check - did you mean to include these setup script updates in this PR?

Reply:
Yes, I needed this while testing and didn't think it was worth making a separate PR for it.